openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -216,16 +217,40 @@ def _filter_fields(self, field_whitelist, serialized_account):
         return visible_serialized_account
 
 
+class UserAccountDisableHistorySerializer(serializers.ModelSerializer):
+    """
+    Class that serializes User account disable history
+    """
+
+    disabled_by = serializers.SerializerMethodField()
+
+    class Meta(object):
+        model = UserAccountDisableHistory
+        fields = ("created", "comment", "disabled", "disabled_by")
+
+    def get_disabled_by(self, disable_history):
+        return disable_history.by.username
+
+
 class AccountUserSerializer(serializers.HyperlinkedModelSerializer, ReadOnlyFieldsSerializerMixin):
     """
     Class that serializes the portion of User model needed for account information.
     """
+    disable_history = serializers.SerializerMethodField()
```

Comment:
Why can't the following work? 
`disable_history = UserAccountDisableHistorySerializer(many=True)`

Reply:
disable_history is not a model field. And the history needs to be filter by user as well